docs/installation_guide.md

```diff
@@ -117,7 +127,7 @@ may not be able to log in after the upgrade.
 
 By default, Harbor stores images on your local filesystem. In a production environment, you may consider 
```

Comment:
I believe this can be configured in harbor.cfg.

Reply:
fixed, add parameters for registry storage